assets/javascript/copy.js

```diff
@@ -19,3 +19,13 @@ codes.forEach((code) => {
 }); 
 
 let clipboard = new ClipboardJS('.btn-copy');
+clipboard.on('success', function(e) {
+    e.clearSelection();
+    e.trigger.className = e.trigger.className.replace("fa-clipboard", "fa-check");
+    e.trigger.setAttribute("title", "Copied!")
+
+    setTimeout(function() {
+        e.trigger.className = e.trigger.className.replace("fa-check", "fa-clipboard");
+        e.trigger.setAttribute("title", "Copy to clipboard")
+    }, 1000);
```

Comment:
I think this can wait a bit longer (maybe 3s)?

Reply:
aaaah, GH showed your comment when I clicked the Merge button.

Reply:
I pushed an update with 2 seconds, I think it's a good compromise.